src/app/core/login/login.component.ts

```diff
@@ -5,6 +5,7 @@ import { Component, OnInit } from '@angular/core';
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.css']
 })
+
```

Comment:
yes, all you javascript related part will go here. In angular we follow typescript and hence you might want to look for typescript equivalents of your code.